src/ResourceManagement/BatchAI/Microsoft.Azure.Management.BatchAI.Fluent.csproj

```diff
@@ -0,0 +1,52 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <PackageId>Microsoft.Azure.Management.BatchAI.Fluent</PackageId>
+        <Version>1.3.1</Version>
```

Comment:
>1.3.1 [](start = 17, length = 5)

please change this to 1.3.0 - we do the version bump before release for all the packages

Reply:
On it. I had just copied the version number being used in Java.